source/process/release-process.md

```diff
@@ -268,23 +284,34 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
 
-1. Build:  
+1. Dev:
+    - Verify hashes and GPG signatures are correct, once build is cut
+    - Update ci-linux-mysql-prev to 4.0.3
```

Comment:
Isn't ci-linux-mysql-prev being left on the current version so that once the new version is released, we still have a server to test on that is on the previous release version?

Reply:
I'm not sure - @jasonblais can you help edit / confirm?

Reply:
Ah yes, Linda actually submitted a PR for this yesterday: https://github.com/mattermost/docs/pull/1557

You can take this part out @amyblais

Reply:
Ok, fixed.